src/themes/mainTheme.ts

```diff
@@ -138,32 +139,42 @@ export const mainTheme = createTheme(
           },
         },
       },
+      MuiStepIcon: {
+        styleOverrides: {
+          root: {
+            fill: Color.StepperBlue,
+            opacity: 0.6,
+            '&.Mui-active': {
+              opacity: 1,
+            },
+            '&.Mui-error': {
+              fill: Color.ErrorMain,
+            },
+          },
+        },
+      },
       MuiStepLabel: {
         styleOverrides: {
           label: {
             opacity: 0.6,
             textTransform: 'uppercase',
             fontSize: '1rem',
-            color: Color.PrimaryMain,
+            color: Color.StepperBlue,
             '&.Mui-active': {
-              color: Color.PrimaryMain,
+              color: Color.StepperBlue,
               fontWeight: 600,
+              borderBottom: '0.06rem solid',
+              mb: '-0.06rem',
```

Comment:
denne er for å kompensere for borderBottom? Kanskje legge til det som en kommentar i så fall? Ser for meg dette er noe jeg fort kan fjerne en dag fordi det ved første øyekast ser rart ut og vanskelig å se hva den gjør